src/tests/end-to-end/test_simple-test.py

```diff
@@ -0,0 +1,82 @@
+from pathlib import Path
+import tempfile as tpf
+from distutils.dir_util import copy_tree
+
+import subprocess
+
+import numpy as np
+
+def find_integrity_path(output_dir):
+    op = []
+    for path in Path(output_dir).rglob('checkIntegrity.txt'):
+        op.append(path)
+    assert len(op) == 1
+    return op[0]
+
+def get_interity_check_values(output : Path) -> np.array :
```

Comment:
function **get_interity_check_values(...)** should be renamed **get_integrity_check_values(...)** (a **g** is missing).
This function does not seem to be called.

Reply:
done x2